docs/tutorials/simulations/sbm.ipynb

```diff
@@ -141,6 +141,135 @@
    "source": [
     "heatmap(G, title='Weighted SBM Simulation')"
    ]
+  },
```

Comment:
can you add this to its own notebook, instead of the same one we have currently?

Reply:
Yes for sure, will be done!